app/script/view_model/WindowTitleViewModel.coffee

```diff
@@ -35,7 +35,8 @@ class z.ViewModel.WindowTitleViewModel
 
       @conversation_repository.conversations_unarchived().forEach (conversation_et) ->
         if conversation_et.type() isnt z.conversation.ConversationType.CONNECT
```

Comment:
We can move all of this into one line:

`if conversation_et.number_of_unread_messages() and not conversation_et.is_request() and not conversation_et.is_muted()`